two_points_on_the_same_branch.sh

```diff
@@ -208,13 +208,20 @@ old_coq_commit_long="$COQ_HASH_LONG"
 # --------------------------------------------------------------------------------
 # Measure the compilation times of the specified OPAM packages in both switches
 
+# Sort the opam packages
+sorted_coq_opam_packages=$("${program_path}/sort-by-deps.sh" ${coq_opam_packages})
```

Comment:
Nit: would it be worth maybe shadowing `coq_opam_packages` here?

Reply:
Sure, I can do that